slam-libraries/viam-cartographer/src/slam_service/config.cc

```diff
@@ -94,6 +102,21 @@ void ParseAndValidateConfigParams(int argc, char** argv,
     slamService.data_rate_ms = std::chrono::milliseconds(FLAGS_data_rate_ms);
     slamService.map_rate_sec = std::chrono::seconds(FLAGS_map_rate_sec);
 
+    // TODO: Remove once PR #1689 is submitted
+    // https://github.com/viamrobotics/rdk/pull/1689 This will allow intergation
+    if (FLAGS_delete_processed_data.empty()) {
+        slamService.delete_processed_data = false;
+        LOG(ERROR) << "HI\n";
+    } else {
+        LOG(ERROR) << "HO\n";
+        slamService.delete_processed_data =
+            !(FLAGS_delete_processed_data == "false");
+    }
+    // slamService.delete_processed_data = FLAGS_delete_processed_data;
+    // if (slamService.offline_flag) {
+    //     slamService.delete_processed_data = false;
```

Comment:
This can be an error, since rdk should never pass this combination of parameters.